### DISCUSSION TITLE
Model tidal bisa kebentuk dalam machine learning di Python

### DISCUSSION THREAD

**Discussion by mosesnugraha** [teamalgoritma/community#784] [Q&A] [answered]
Selamat siang ini saya mau buat modelnya tapi Kenapa dibilang tipe datanya harus diubah ke float yaa padahal kolom tersebut tipe Catania suda float

<img width="1440" alt="Screen Shot 2023-10-17 at 13 04 36" src="https://github.com/teamalgoritma/community/assets/123171973/ca9279ae-b1b0-47f8-ae64-7310882dba2d">

**Comment by jafarhf1** [comment, marked as answer]
Dear Pak @mosesnugraha ,

Kalau dilihat berdasarkan errornya sepertinya di data yang Bapak miliki terdapat format string yang tidak bisa diubah menjadi float yaitu "O3" sepertinya itu menggunakan huruf O bukan angka 0. Boleh dicek kembali pak untuk datanya. 

Semoga membantu.

Best regards,
Team Algoritma
